web/src/lib/domain-logic/shouldShowDisclaimerForProfileNotSubmittingData.test.ts

```diff
@@ -0,0 +1,102 @@
+import { shouldShowDisclaimerForProfileNotSubmittingData } from "@/lib/domain-logic/shouldShowDisclaimerForProfileNotSubmittingData";
+import { BusinessPersona, businessPersonas } from "@businessnjgovnavigator/shared/profileData";
+import {
+  generateBusiness,
+  generateFormationData,
+  generateProfileData,
+} from "@businessnjgovnavigator/shared/test";
+
+import { IsAuthenticated } from "@/lib/auth/AuthContext";
+
+const nonOwningPersonas: BusinessPersona[] = ["STARTING", "FOREIGN"];
+
+describe("shouldShowDisclaimerForProfileNotSubmittingData", () => {
+  it("returns true if business is not defined", () => {
+    expect(shouldShowDisclaimerForProfileNotSubmittingData(undefined, IsAuthenticated.TRUE)).toBe(true);
+    expect(shouldShowDisclaimerForProfileNotSubmittingData(undefined, IsAuthenticated.FALSE)).toBe(true);
+  });
+
+  it("returns true if isAuthenticated is UNKNOWN", () => {
+    const business = generateBusiness({
+      formationData: generateFormationData({
+        completedFilingPayment: false,
+      }),
+      profileData: generateProfileData({
+        dateOfFormation: undefined,
```

Comment:
Pebble: based on how your conditional is written, `isFormedOutsideNavigator(business)` is causing the conditional to fail. 

I'd update the `dateOfFormation` and `businessPersona` so the first two conditionals are true (see below)
```
    isFormedOutsideNavigator(business) ||
    business.profileData.businessPersona === "OWNING" ||
    isAuthenticated === IsAuthenticated.FALSE
```